tweetapi/models.py

```diff
@@ -21,6 +21,21 @@ def __str__(self):
         return self.content
 
 
-        
 
+class CommentModel(models.Model):
+    owner = models.ForeignKey(User,on_delete=models.CASCADE, related_name="comment")
+    tweet = models.ForeignKey(TweetModel,related_name="comment",on_delete=models.CASCADE)
```

Comment:
related names should be plural for foreign keys

Reply:
ok sir, got it

Reply:
fixed sir